heareval/tasks/pipeline.py

```diff
@@ -237,23 +220,41 @@ def get_subsample_key(df: DataFrame) -> Series:
         assert "slug" in df, "slug column not found in the dataframe"
         return df["slug"].apply(str).apply(filename_to_int_hash)
 
-    def get_process_metadata(self) -> pd.DataFrame:
+    @staticmethod
+    def get_stratify_key(df: DataFrame) -> Series:
         """
-        Return a dataframe containing the task metadata for this
-        entire task.
+        CURRENTLY UNUSED.
```

Comment:
Comment on why? -- This stratification works well for multi-class scene based tasks, but becomes more complicated for multilabel and event based tasks, so this is unused for the time being until we can figure that out. Or something like that.

Reply:
@jorshi I just straight up remove it later so I'll merge it. Maybe worth a talk in our summary paper